native/android/build.gradle

```diff
@@ -30,13 +30,15 @@ buildscript {
     }
     repositories {
         google()
+        mavenCentral()
+        // TODO Remove. Still required for some libraries.
```

Comment:
Why is this needed?

Reply:
See https://github.com/maplibre/maplibre-gl-native/issues/92, jcenter seems to be needed until mapbox-sdk-turf v5.12.0 while we are currently on v5.3.0. If I understood @f1sh1918 correctly we are not able to upgrade yet though, so we still need jcenter.